dts/bindings/timer/ti,am654-dmtimer.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI dual-mode Timer
+
+compatible: "ti,am654-dmtimer"
+
```

Comment:
Looks like the compatible also doesn't match what is in upstream bindings/DT

Reply:
Good idea, I'll re-work this so the bindings match.